frontend/src/components/editor/navigation/navigation.ts

```diff
@@ -326,7 +404,11 @@ export function useCellNavigationProps(
     },
   });
 
-  return mergeProps(focusWithinProps, keyboardProps);
+  return mergeProps(focusWithinProps, keyboardProps, {
+    "data-selected": isSelected,
+    className:
+      "data-[selected=true]:ring-1 data-[selected=true]:ring-[var(--blue-10)] data-[selected=true]:ring-offset-1",
```

Comment:
single selection is `--blue-7`, `--blue-8` looks good for multi (radix recommends 6-8 separators and borders )

```suggestion
      "data-[selected=true]:ring-1 data-[selected=true]:ring-[var(--blue-8)] data-[selected=true]:ring-offset-1",
```